css/common.css

```diff
@@ -0,0 +1,415 @@
+@import url("font.css");
+.container {
+  margin: 0;
+  display: grid;
+  grid-template-columns: 16.375em 1fr;
+  grid-template-rows: 5.813em;
+  grid-template-areas:
+    "sidebar user-options-section"
+    "sidebar main";
+}
+body {
+  margin: 0;
+  font-family: "Noto Sans", sans-serif;
+  font-family: "Poppins", sans-serif;
+}
+span {
+  margin: 0;
+  padding: 0;
+}
+.sidebar-handle {
+  position: absolute;
+  right: -0.9em;
+  top: 2em;
+}
+
+.sidebar a {
+  text-decoration: none;
+  color: #6b7075;
+}
+.sidebar {
+  grid-area: sidebar;
+  width: 16.375rem;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 0px 0px 20px 1px rgba(201, 194, 194, 0.1);
+  justify-content: space-between;
+  color: #6b7075;
+  position: fixed;
+  height: 100vh;
+}
+.container ul {
+  list-style-type: none;
+}
+.logo {
+  padding-left: 1em;
+  padding-top: 0.938em;
+  font-size: 2em;
+  margin: 0;
+}
+.all-dashboard {
+  margin-top: 1.5em;
+}
+.all-dashboard-heading {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  margin-bottom: 5px;
+}
+.all-dashboard-heading-line {
+  background-color: #6b7075;
+  height: 1px;
+  width: 1rem;
+  opacity: 0.3;
+  margin: 0;
+}
+
+.all-dashboard p {
+  font-size: 0.75em;
+  vertical-align: middle;
+  opacity: 0.5;
+  padding-left: 6.5%;
+}
+
+.all-dashboard ul {
+  margin: 0;
+  padding: 0;
+}
+
+.all-dashboard ul li {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-left: 5.4%;
+  padding-top: 4.4%;
+  padding-bottom: 4.4%;
+  padding-left: 6%;
+  padding-right: 0%;
+  border-radius: 6px;
+  font-size: small;
+  font-weight: 600;
+}
+.highlight {
+  position: absolute;
+  right: 0;
+  top: 2.5em;
+  margin: 0;
+  padding: 0;
+  width: 4px !important;
+}
+.role-management-dashboard {
+  position: relative;
+}
+
+.dashboard-selection-highlight {
+  float: inline-end;
+  position: relative;
+  top: -2.8em;
+  background-color: #f44848;
+  width: 4px !important;
+  margin: 0;
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+  height: 2.8em;
+}
+
+.all-dashboard-selected-menu li {
+  width: 81.5%;
+  border-radius: 7px;
+  background-color: #fde9e9;
+  padding: 4% 0 4% 5.6% !important;
+  margin-left: 1.31em;
+  margin-top: -2px;
+  color: #f44848;
+}
+.all-dashboard ul li img {
+  padding-right: 6%;
+}
+.all-dashboard-selected-menu {
+  display: flex;
+}
+.all-dashboard-selected-menu a {
+  color: #f76f6f;
+}
+.roles-selected-menu a {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+.roles-selection-menu a {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+.roles-selected-menu a div {
+  margin-right: 5.2em;
+}
+.roles-selection-menu a div {
+  width: 15.2em;
+}
+.all-dashboard ul li img {
+  margin-right: 0.7em;
+}
+ul li a {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+ul li a img {
+  margin-right: 5px;
+}

```

Comment:
Try to make them more specific to the elements you are targeting to avoid unintended styling conflicts.

Reply:
thanks for highlighting it I have changed it to be more specific so that css to apply only the dashboard UL  as it needs to